src/FSharpLint.Core/Rules/Conventions/UsedUnderscorePrefixedElements.fs

```diff
@@ -0,0 +1,56 @@
+module FSharpLint.Rules.UsedUnderscorePrefixedElements
+
+open System
+open FSharpLint.Framework
+open FSharpLint.Framework.Suggestion
+open FSharp.Compiler.Syntax
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.Rules
+
+let private checkUsedIdent (previousIdent: Ident) (body: SynExpr) =
+    match body with
+    | SynExpr.Sequential(debugPointAtSequential, isTrueSeq, synExpr, expr2, range) ->
```

Comment:
@realmarv isTrueSeq (and others) is not being used, therefore should be named `_isTrueSeq`

Reply:
Done